st2common/st2common/bootstrap/runnersregistrar.py

```diff
@@ -400,6 +400,30 @@
 
     # Experimental runners below
     {
+        'name': 'announcement',
+        'aliases': [],
+        'description': 'A runner for emitting an announcement event on the stream.',
+        'enabled': True,
+        'runner_parameters': {
+            'experimental': {
```

Comment:
Any reason you believe this ack is important? I guess we could simply not document this and by obfuscation it would be experimental.